browser/src/robocorp/browser/__init__.py

```diff
@@ -1,5 +1,4 @@
-"""
-Main module for doing browser automation with Playwright.
+"""Main module for doing browser automation with Playwright.
 
 This library can be made available by pinning
 ![`robocorp-browser`](https://img.shields.io/pypi/v/robocorp-browser?label=robocorp-browser)
```

Comment:
While at it, we can turn this badge into a clickable URL that will direct you to the PyPI package, I think that wrapping it with `[<content>](<url>)` might do the job, but you should try first.